v2/googlecloud-to-neo4j/src/main/java/com/google/cloud/teleport/v2/neo4j/database/Neo4jConnection.java

```diff
@@ -141,23 +141,21 @@ private void deleteData() {
 
   private void dropSchema(Neo4jCapabilities capabilities) {
     try (var session = getSession()) {
-      if (capabilities.hasConstraints()) {
-        LOG.info("Dropping constraints");
-        var constraints =
-            session
-                .run(
-                    "SHOW CONSTRAINTS YIELD name",
-                    Map.of(),
-                    databaseResetMetadata("show-constraints"))
-                .list(r -> r.get(0).asString());
-        for (var constraint : constraints) {
-          LOG.info("Dropping constraint {}", constraint);
-
-          runAutocommit(
-              String.format("DROP CONSTRAINT %s", CypherPatterns.sanitize(constraint)),
-              Map.of(),
-              databaseResetMetadata("drop-constraint"));
-        }
+      LOG.info("Dropping constraints");
```

Comment:
why don't we check `capabilities.hasConstraints()` anymore?

Reply:
because unique constraints are always available